ocs_ci/ocs/bucket_utils.py

```diff
@@ -1954,12 +1954,23 @@ def create_aws_bs_using_cli(
     )
 
 
-def expire_objects_in_bucket(bucket_name):
+def upload_bulk_buckets(s3_obj, buckets, amount=1, object_key="obj-key-0", prefix=None):
+    """
+    Upload object to the buckets
+    """
```

Comment:
Docstring elaboration to improve clarity:
```suggestion
    """
    Upload given objects with sequential keys to multiple buckets.

    Args:
    - s3_obj: obc/mcg object
    - buckets (list): list of bucket names to upload to
    - amount (int, optional): number of objects to upload per bucket
    - object_key (str, optional): base object key
    - prefix (str, optional): prefix for the upload path

    """
```